### ISSUE TITLE
Cannot subscribe to librem.one user

### ISSUE THREAD

**Issue by MaximilianAst** [nextcloud/social#573]
### Steps to reproduce
1. Go to 'https://social.librem.one/@lunduke'
2. Click on 'Follow'
3. Enter your id (i.e. @user@nextcloud.hostname.com)
4. See error RequestContentException (on https://nextcloud.hostname.com/index.php/apps/social/ostatus/follow/lunduke%40librem.one)

### Expected behaviour
I should be subscribed to @lunduke@librem.one

### Actual behaviour
If I subscribe a user at librem.one, which is a mastodon fork. I get an `RequestContentException` error:
``` json
{
"status": -1,
"exception": "OCA\\Social\\Exceptions\\RequestContentException",
"message": "404 - {\"protocol\":\"https\",\"host\":\"librem.one\",\"url\":\"/.well-known/webfinger\",\"timeout\":10,\"type\":0,\"data\":{\"resource\":\"acct:lunduke@librem.one\"}}"
}
```

## Server configuration detail

**Operating system:** Linux 3.13.0-95-generic #142-Ubuntu SMP Fri Aug 12 17:00:09 UTC 2016 x86_64

**Webserver:** Apache/2.4.39 (Ubuntu) (apache2handler)

**Database:** mysql 5.7.26

**PHP version:** 

7.2.19-1+ubuntu16.04.1+deb.sury.org+1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, apache2handler, mysqlnd, PDO, xml, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imagick, intl, json, exif, mysqli, pdo_mysql, Phar, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Zend OPcache

**Nextcloud version:** 16.0.1 - 16.0.1.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Nextcloud 15.0.8

**Where did you install Nextcloud from:** .tar.bz2

<details><summary>Signing status</summary>

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.2.0
 - activity: 2.9.1
 - admin_audit: 1.6.0
 - apporder: 0.7.1
 - audioplayer: 2.7.0
 - bookmarks: 1.0.5
 - bruteforcesettings: 1.3.0
 - calendar: 1.7.0
 - checksum: 0.4.3
 - cloud_federation_api: 0.2.0
 - comments: 1.6.0
 - contacts: 3.1.1
 - cospend: 0.1.0
 - data_request: 1.3.0
 - dav: 1.9.2
 - deck: 0.6.2
 - drawio: 0.9.3
 - external: 3.3.0
 - federatedfilesharing: 1.6.0
 - federation: 1.6.0
 - files: 1.11.0
 - files_markdown: 2.0.6
 - files_pdfviewer: 1.5.0
 - files_readmemd: 1.0.4
 - files_rightclick: 0.13.0
 - files_sharing: 1.8.0
 - files_texteditor: 2.8.0
 - files_trashbin: 1.6.0
 - files_versions: 1.9.0
 - files_videoplayer: 1.5.0
 - firstrunwizard: 2.5.0
 - flowupload: 0.1.0
 - gallery: 18.3.0
 - gpxpod: 3.0.3
 - groupfolders: 4.0.2
 - issuetemplate: 0.5.0
 - keeweb: 0.5.1
 - logreader: 2.1.0
 - lookup_server_connector: 1.4.0
 - metadata: 0.9.0
 - news: 13.1.6
 - nextcloud_announcements: 1.5.0
 - notes: 2.6.0
 - notifications: 2.4.1
 - oauth2: 1.4.2
 - password_policy: 1.6.0
 - passwords: 2019.6.0
 - phonetrack: 0.5.1
 - polls: 0.10.2
 - privacy: 1.0.0
 - provisioning_api: 1.6.0
 - radio: 0.6.5
 - rainloop: 6.0.2
 - recommendations: 0.4.0
 - serverinfo: 1.6.0
 - sharebymail: 1.6.0
 - social: 0.2.3
 - spreed: 6.0.1
 - support: 1.0.0
 - systemtags: 1.6.0
 - tasks: 0.10.1
 - theming: 1.7.0
 - twofactor_backupcodes: 1.5.0
 - updatenotification: 1.6.0
 - viewer: 1.0.0
 - workflowengine: 1.6.0
Disabled:
 - encryption
 - files_external
 - mail
 - ocdownloader
 - socialsharing_email
 - survey_client
 - twofactor_totp
 - twofactor_u2f
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nextcloud.hostname.com"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/nextcloud.hostname.com",
    "dbtype": "mysql",
    "version": "16.0.1.1",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "maintenance": false,
    "theme": "",
    "loglevel": 2,
    "updater.release.channel": "stable",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "465",
    "mail_smtpsecure": "ssl",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "app_install_overwrite": [
        "data_request"
    ]
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** false

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Chrome/74.0.3729.131

**Operating system:** Windows 1803

## Logs

<details><summary>Web server error log</summary>

```

```
</details>

<details><summary>Nextcloud log</summary>

```

```
</details>

<details><summary>Browser log</summary>

</details>

**Comment by daita** [comment]
Thanks for your report;

would be nice to implement it in both ways:

- checking host-meta on remote request 
- helping to implement on local instance (#505)

**Comment by MaximilianAst** [comment]
I found a workaround for librem.one. Copy the URI from the popup-window to a new tab. Now add social in the URI:

user%40librem.one to user%40social.librem.one

https://nextcloud.hostname.com/index.php/apps/social/ostatus/follow/user%40social.librem.one

**Comment by daita** [comment]
Please confirm issue is fixed in 0.2.5

**Comment by r3pek** [comment]
I'm seeing the behaviour (well, kinda) on https://mastodon.social/@x0rz and https://botsin.space/@duckduckgo

Reply status is (after clicking "follow"):

status | -1
-- | --
exception | "OCA\\Social\\Exceptions\\ItemUnknownException"
message | ""

**Comment by daita** [comment]
hmm x0rz ? from #linuxfr ?

**Comment by r3pek** [comment]
> hmm x0rz ? from #linuxfr ?

Yes, I think that's the one.

**Comment by daita** [comment]
it's been a while ...

**Comment by r3pek** [comment]
> it's been a while ...

well, it's not me :) I just follow him there ;)

**Comment by daita** [comment]
i understood that ;-) 
still reminds me how old we are !

**Comment by r3pek** [comment]
Don't even tell me about it ;) (3 kids aren't easy :angel: )

**Comment by MaximilianAst** [comment]
@daita 

> Please confirm issue is fixed in 0.2.5

Just Updated to 0.2.6 and it's fixed
